controllers/index.js

```diff
@@ -0,0 +1,10 @@
+const {
+    selectCategories
+} = require("../models");
+
+exports.getCategories = (req, res) => {
+    
+  selectCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
```

Comment:
I would add a catch here just incase there is a problem with this function you know its going to be handled by your error handling chain 👍